examples/with-one-login-many-subdomains/frontend/src/App.js

```diff
@@ -82,7 +91,7 @@ function App() {
                     <div className="fill">
                         <Routes>
                             {window.location.origin === getAuthDomain() ? (
-                                <Route path="/auth" element={<AuthPage />} />
+                                <Route path="/auth/*" element={<AuthPage />} />
```

Comment:
In AuthPage, we change the `if` statement to `if (tenantId !== undefined || path !== getWebsiteBasePath()) {`